MEMBER_REPRESENTATION.md

```diff
@@ -1,27 +1,59 @@
-# Member Representation
+# OpenJS Foundation Representatives
 
-The following document lists OpenJS Foundation members and their representation to any standards organizations:
+The following document lists OpenJS Foundation's memberships and liaison relationships, along with the community members designated to participate on behalf of the Foundation:
+
+* [Ecma International]
+  * [TC39] 
+    * Guy Bedford
+    * Jordan Harband
+    * Lea Verou
+    * Richard Gibson
+  * [TC53] 
+    * Donovan Buck
+  * General Assembly Representative
+    * Jory Burson
 * [Open Source Initiative]
-  * Tobie Langel
-* [TC39]
-  * Guy Bedford
-  * Jordan Harband
-  * Jory Burson (Ecma GA Representative)
-  * Lea Verou
-  * Richard Gibson
-* [TC53]
-  * Donovan Buck
+  * Tobie Langel (Delegate)
 * [Unicode MessageFormat Working Group]
   * Eemeli Aro
 * [W3C]
-  * Brian Kardell
-  * Jory Burson (AC)
-  * Michael Champion
-  * Tobie Langel
+  * [Web-interoperable Runtimes Community Group]
+    * Jory Burson
+    * Robin Ginn
```

Comment:
```suggestion
    * Robin Ginn
    * Jordan Harband
```
i participate in wintercg whenever i can

Reply:
Me too! `* Ethan Arrowood`